src/backend/allocator/dmabuf.rs

```diff
@@ -397,11 +397,11 @@ impl DmabufSource {
         ];
         for (idx, handle) in dmabuf.handles().enumerate() {
             // SAFETY: This is stored together with the Dmabuf holding the owned file descriptors
-            let fd = handle.as_raw_fd();
+            let fd = handle.as_fd();
             if matches!(
                 poll::poll(
                     &mut [poll::PollFd::new(
-                        fd,
+                        &fd,
```

Comment:
`PollFd::new` takes a reference of something implementing `AsFd` and `dmabuf.handles()` return an iterator over `BorrowedFd`. So I guess we could directly pass `handle` here?